SofascoreAcademy/SofascoreAcademy/ScoreLabel.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  LeagueNameView.swift
+//  SofascoreAcademy
+//
+//  Created by Akademija on 12.03.2024..
+//
+
+import Foundation
+import SnapKit
+import UIKit
+import SofaAcademic
+
+class ScoreLabel: BaseView {
+    
+    let score: Int
+    let scoreString: String
+    let matchStatus: matchStatus
+    
+    let scoreLabel = UILabel()
+    let textColor: UIColor
+
+    init(score: Int, matchStatus: matchStatus) {
```

Comment:
Ovo primjecujem i tu i na TimeStatusStackView, ali ti ostavljam komentar samo ovdje pa primjeni na oba mjesta.
Dobro si napravio i ovo je super sad ovako za vjezbu ili u slucaju da radis neku komponentu koja je uvijek ista, tj staticka.
Medjutim, kad dodu pravi podaci score i match status je nesto sto ce se mijenjati, tj ti ces trebati azurirati ovu komponentu. Na ovaj nacin je to moguce jedino da ju napravis ponovno. BIlo bi bolje kad bi imao neku fju za azuriranje dostupnu van scope-a ove klase (public) u kojoj bi postavljao score i match status.

Reply:
Sto cemo s ovim?

Reply:
Ako me pitas za score, postavio sam da se score dinamicki mijenja u MatchView, jel to ok?